### PR TITLE
Fix S3 backup job

### PR DESCRIPTION
**Pull request by conbrad**
- Repoints backup job to the crunchy cluster
- Removes unused parameters
# Test Links:
[Landing Page](https://wps-pr-3397.apps.silver.devops.gov.bc.ca/)
[MoreCast 2.0](https://wps-pr-3397.apps.silver.devops.gov.bc.ca/morecast-2)
[Percentile Calculator](https://wps-pr-3397.apps.silver.devops.gov.bc.ca/percentile-calculator)
[MoreCast](https://wps-pr-3397.apps.silver.devops.gov.bc.ca/morecast)
[C-Haines](https://wps-pr-3397.apps.silver.devops.gov.bc.ca/c-haines)
[FireBat](https://wps-pr-3397.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator)
[FireBat bookmark](https://wps-pr-3397.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator?s=266&f=c5&c=NaN&w=20,s=286&f=c7&c=NaN&w=16,s=1055&f=c7&c=NaN&w=NaN,s=305&f=c7&c=NaN&w=NaN,s=344&f=c5&c=NaN&w=NaN,s=346&f=c7&c=NaN&w=NaN,s=328&f=c7&c=NaN&w=NaN,s=1399&f=c7&c=NaN&w=NaN,s=334&f=c7&c=NaN&w=NaN,s=1082&f=c3&c=NaN&w=NaN,s=388&f=c7&c=NaN&w=NaN,s=309&f=c7&c=NaN&w=16,s=306&f=c7&c=NaN&w=NaN,s=1029&f=c7&c=NaN&w=NaN,s=298&f=c7&c=NaN&w=NaN,s=836&f=c7&c=NaN&w=NaN,s=9999&f=c7&c=NaN&w=NaN)
[Auto Spatial Advisory (ASA)](https://wps-pr-3397.apps.silver.devops.gov.bc.ca/auto-spatial-advisory)
[HFI Calculator](https://wps-pr-3397.apps.silver.devops.gov.bc.ca/hfi-calculator)

### CODE REVIEW

**Review by dgboss — APPROVED** [review]
Nice!

File: openshift/scripts/oc_provision_backup_s3_postgres_cronjob.sh [hunk 1]
```diff
@@ -29,11 +29,11 @@ PROJ_TARGET="${PROJ_TARGET:-${PROJ_DEV}}"
 # Prepare variables for backups
 JOB_NAME="backup-postgres-${APP_NAME}-${SUFFIX}"
 IMAGE_NAMESPACE=${PROJ_TOOLS}
-CLUSTER_NAME="patroni-${APP_NAME}-${SUFFIX}"
+CLUSTER_NAME="wps-crunchydb-${SUFFIX}"
 
 OC_PROCESS="oc -n ${PROJ_TARGET} process -f ${TEMPLATE_PATH}/backup-s3-postgres-cronjob.yaml \
-    -p DATABASE_SERVICE_NAME=patroni-${APP_NAME}-${SUFFIX}-leader \
     -p DATABASE_DEPLOYMENT_NAME=wps-global \
+    -p CRUNCHYDB_USER=wps-crunchydb-${SUFFIX}-pguser-wps-crunchydb-${SUFFIX} \
```
Comment (brettedw) [inline openshift/scripts/oc_provision_backup_s3_postgres_cronjob.sh:36]: Does it still need to be passed in here if its specified in the yaml env below? More of a curiosity question

Comment (conbrad) [reply openshift/scripts/oc_provision_backup_s3_postgres_cronjob.sh:36]: Yes, this passes in the value as a parameter to the yaml file. When `oc process` is called these values are filled in.

Comment (brettedw) [reply openshift/scripts/oc_provision_backup_s3_postgres_cronjob.sh:36]: Oh I see what's happening, thanks. Monday brain no work so good

**Pull request merged**